routes/middlewares/festival/lol/get/getLol.ts

```diff
@@ -0,0 +1,35 @@
+import { NextFunction, Request, Response } from 'express';
+
+import CustomError from '@Middleware/error/customError';
+import LolTeam from '@Model/lolTeam.model';
+import LolTeamMember from '@Model/lolTeamMember.model';
+import LolVote from '@Model/lolVote.model';
+
+const getLol = async (req: Request, res: Response, next: NextFunction) => {
+  try {
+    const result: LolTeam[] = await LolTeam.findAll({
+      include: [
+        {
+          model: LolTeamMember,
+          attributes: ['pk', 'team_pk', 'name', 'studentId', 'leader'],
+        },
+        {
+          model: LolVote,
+          attributes: ['pk', 'team_pk', 'user_pk', 'createdAt'],
+        },
+      ],
+    });
+
+    res.json({
+      success: true,
+      data: {
+        result,
```

Comment:
```suggestion
        team,
```
응답값의 Key값도 상당히 중요함, 제대로 된 값을 던져줘야 프론트에서 식별하기 편함